pkgs/development/compilers/ispc/default.nix

```diff
@@ -30,8 +30,7 @@ stdenv.mkDerivation rec {
 
   inherit testedTargets;
 
-  # needs 'transcendentals' executable, which is only on linux
```

Comment:
I don't know what this executable is, but tests were failing for me precisely because the targets weren't working. I would expect that to fail the compilation, but apparently not